nettacker/config.py

```diff
@@ -102,6 +102,7 @@ class PathConfig:
     tmp_dir = CWD / ".data/tmp"
     web_static_dir = PACKAGE_PATH / "web/static"
     user_agents_file = PACKAGE_PATH / "lib/payloads/User-Agents/web_browsers_user_agents.txt"
+    compare_results_base_path = CWD / "results"
```

Comment:
@Captain-T2004 this is incorrect - compare_results_base_path should be the same as results_dir (see line 101) and as a results I get error
![Screenshot 2024-09-13 at 23 16 29](https://github.com/user-attachments/assets/6c3948f2-bc39-4dc8-81ff-cc7e9982b8b5)

Reply:
Yes, I will change that back to results_dir, I actually made a separate directory called /results in my Nettacker directory but it was not commited as it was an empty dir. I did this because after adding the path sanitization the user can only save files in the allowed results path with a modifiable file name. This way I though it would be better to find the results fairly easily. I will revert it back but do you think I should alter the path sanitization to include more paths? not just the allowed results_dir path.

Reply:
@Captain-T2004  Please revert back - There is no need to have additional 'results' folder - Nettacker already has one official results folder in the '.data/' for all the results. In order to make comparison results easier you might consider using a file name like this 'results_compare_{scan_id}' - this way it will be easy to find.